examples/spring-boot-on-openshift-example/src/main/java/io/dekorate/example/sbonopenshift/Main.java

```diff
@@ -15,15 +15,11 @@
  */
 package io.dekorate.example.sbonopenshift;
 
-import io.dekorate.kubernetes.annotation.Env;
 import io.dekorate.openshift.annotation.OpenshiftApplication;
-import io.dekorate.option.annotation.JvmOptions;
-import io.dekorate.option.annotation.SecureRandomSource;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@OpenshiftApplication(buildEnvVars = @Env(name="foo", value = "bar"))
-@JvmOptions(server = true, preferIPv4Stack = true, secureRandom = SecureRandomSource.NonBlocking)
```

Comment:
Was this example suppose to show the use of these? If not, does it still make sense to keep it?

Reply:
IMHO, our examples are bloated with stuff that should be part of integration tests instead, so these need to go.

Reply:
Gotcha, thanks